utils.py

```diff
@@ -0,0 +1,96 @@
+import mysql.connector
```

Comment:
The utils file should be renamed to db_utils. 

reasons: 
utils: Imported and used whenever general-purpose functions are needed.
db_utils: Imported and used when database interactions are required.